node/tessel-export.js

```diff
@@ -648,6 +655,13 @@ Tessel.I2C.prototype.transfer = function(txbuf, rxlen, callback) {
   this._port.uncork();
 };
 
+Tessel.I2C.prototype.disable = function() {
+  // Tell the coprocessor to disable this interface
+  this._port._simple_cmd([CMD.CMD_DISABLE_I2C]);
+  // Note that the bus is not enabled
+  this._port.I2C.enabled = false;
+};
+
```

Comment:
Someone may want to disable the interface when they are done with I2C so the pins don't stay pulled high.